src/js/tabs/btc.controller.js

```diff
@@ -27,6 +27,10 @@ BtcTab.prototype.angular = function (module)
       $scope.showInstructions = !$scope.showInstructions;
     };
 
+    $scope.toggle_bitstamp_instructions = function (){
+        $scope.showBitstampInstructions = !$scope.showBitstampInstructions;
```

Comment:
I know we have to write some custom code / text for different integrations, but can we at least not have 3rd party service names in the code?